### DISCUSSION TITLE
How do I extract the tables from the sqlite file?

### DISCUSSION THREAD

**Discussion by Hemang999** [kyleskom/NBA-Machine-Learning-Sports-Betting#357] [General]
I want to extract the tables from the dataset.sqlite file, view the data and convert it into xlsx format. How do I do that? Everytime I try to create the connection to the sqlite file in python using sqlite3 I always get an error saying there is no table called dataset_2012-24.

**Comment by nealmick** [comment]
It may be useful to try a sqlite db viewer.  This is a type of program similar to excel or google sheets but for sqlite databases.  I think so long as the original database hasnt been corrupted in some way... there are many ways to dump a database.   I highly recommend asking gpt something like how to dump sqlite to csv.  I would recommend trying to get csv working first its kinda a simpler concept, csv is just comma separated variable, as simple as 1,2,3.   xlsx i thinnk may have extra's as far as the header for columns and things like that but its definitely still doable....  also like might be better to just take take the pandas dataframe after its loaded the table and convert that to csv, might be easier.....
